README.md

```diff
@@ -4,33 +4,34 @@
 [![Travis Build Status](https://travis-ci.org/HewlettPackard/oneview-chef.svg?branch=master)](https://travis-ci.org/HewlettPackard/oneview-chef)
 [![Chef Build Status](https://jenkins-01.eastus.cloudapp.azure.com/job/oneview-cookbook/badge/icon)](https://jenkins-01.eastus.cloudapp.azure.com/job/oneview-cookbook/)
 
-Chef cookbook that provides resources for managing OneView.
+Chef cookbook that provides resources for managing HPE OneView.
 
 ## Requirements
 
  - Chef 12.0 or higher
- - HPE OneView 2.0 (may work with other versions too, but no guarantees)
+ - HPE OneView 2.0 or 3.0 (may work with other versions too, but no guarantees)
```

Comment:
How about
> HPE OneView appliance that supports API version 200 or 300 (may work with other versions too, but no guarantees)

This may make it a little more difficult for people that don't know what API version they have, but do know the release number (i.e., 2.0)